sdk/identity/azure-identity/src/managed_identity_source.cpp

```diff
@@ -180,10 +183,17 @@ AppServiceManagedIdentitySource::AppServiceManagedIdentitySource(
 
     url.AppendQueryParameter("api-version", apiVersion);
 
+    // Only one of clientId or resourceId will be set to a non-empty value.
```

Comment:
Do we handle the case where both are non-empty?

Reply:
I wanted to, but the public API is designed to not hit this scenario (user can't pass in both), hence no need to validate explicitly.